src/ResultPaginator.php

```diff
@@ -66,31 +67,15 @@ public function __construct(
      */
     public function search($expression)
     {
-        return new FlatMapIterator(
+        return t\to_iter(
             $this,
-            function (Result $result) use ($expression) {
-                return (array) $result->search($expression);
-            }
+            t\mapcat(function (Result $result) use ($expression) {
+                return (array)$result->search($expression);
+            })
         );
     }
 
     /**
-     * Returns either a single config value or entire paginator's config.
-     *
-     * @param string $key
-     *
-     * @return string|array|null
-     */
-    public function getConfig($key = null)
```

Comment:
I removed this function because I didn't see a need for it (at least yet). It seems like an implementation detail of the iterator leaking through.